Makefile

```diff
@@ -119,16 +132,15 @@ docker-push: ## Push docker image with the manager.
 
 ##@ Deployment
 
+ifndef ignore-not-found
+  ignore-not-found = false
+endif
+
 # Use create due to the annotations limit in apply
 .PHONY: install ## Install CRDs into the K8s cluster specified in ~/.kube/config.
 install: manifests kustomize
 	$(KUSTOMIZE) build config/crd | kubectl create -f -
 
-# Replace CRDs into a cluster
-.PHONY: replace
-replace: manifests kustomize ## Replace CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl replace -f -
-
```

Comment:
This target is not scaffolded by operator sdk, but it might be useful for development purposes. What do you think?

Reply:
Mmmh ok, let's keep it